### PR TITLE
Add Cryptee  #160

### PR DESCRIPTION
**Pull request by woutfeys**
Add Cryptee  #160

### CODE REVIEW

**Comment by tycrek** [comment]
Ok, I really screwed that one up... Trying to figure out how to revert that delete commit

File: README.md [hunk 1]
```diff
@@ -936,4 +936,4 @@ It's a shame that Google, with their immense resources, power, and influence, do
 
 Hopefully this guide can serve as a starting point for those new to privacy, or be a good refresher for the experts.
 
-Good luck and stay safe!
\ No newline at end of file
+Good luck and stay safe!
```
Comment (tycrek) [inline README.md]: No idea why git thinks this is a change, looks identical to me

File: yaml/degoogle.yml [hunk 2]
```diff
@@ -334,6 +334,11 @@ web based products:
       eyes: null
       text: |
         It's an open-source (and offline) document processor using TeX/LaTeX. It's different than Docs or Word but amazing, once you get used to it. Thanks @Glitchy-Tozier
+    - name: Cryptee
+      url: https://crypt.ee/
+      eyes: null
+      text: |
+        Offline version of Cryptee for web. Runs as a progressive web app. Works most modern phone browsers (Chrome, Opera or Firefox on Android; Safari on iOS). Also works in desktop browsers that support PWA (the website lists that only Chrome works, but [others will work](https://github.com/tycrek/degoogle/issues/160#issuecomment-643066320) too.
```
Comment (tycrek) [inline yaml/degoogle.yml:341]: Improved the description

File: README.md (hunk 1 above)
Comment (woutfeys) [reply README.md]: I think because they changed from place in the file

Comment (koenvervloesem) [reply README.md]: One doesn't have a newline at the end of the file, the other one has.

**Pull request merged**